fc/parsing/rdf.py

```diff
@@ -0,0 +1,52 @@
+"""
+RDF handling routines, including parsing the 'oxmeta' ontology.
+"""
+import pkg_resources
+
+import rdflib
+
+from cellmlmanip.rdf import create_rdf_node
+
+
+_ONTOLOGY = None  # The 'oxmeta' ontology graph
+
+OXMETA_NS = 'https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata#'
+BQBIOL_NS = 'http://biomodels.net/biology-qualifiers/'
+
+PRED_IS = create_rdf_node((BQBIOL_NS, 'is'))
+PRED_IS_VERSION_OF = create_rdf_node((BQBIOL_NS, 'isVersionOf'))
+
+
+def get_variables_transitively(model, term):
+    """Return a list of variables annotated (directly or otherwise) with the given ontology term.
+
+    Direct annotations are those variables annotated with the term via the bqbiol:is or
+    bqbiol:isVersionOf predicates.
+
+    However we also look transitively through the 'oxmeta' ontology for terms belonging to the RDF
+    class given by ``term``, i.e. are connected to it by a path of ``rdf:type`` predicates, and
+    return variables annotated with those terms.
```

Comment:
Can you give an example here?

Reply:
I'm not sure how to understand "the RDF class given by ``term``"

Reply:
OK, will try to do that.

Reply:
Done.

Reply:
Thanks!